packages/web-wallet/src/context/WebzjsContext.tsx

```diff
@@ -146,11 +147,7 @@ export const WebZjsProvider = ({ children }: { children: React.ReactNode }) => {
 
   useEffect(() => {
     if (state.error) {
-      const timeout = setTimeout(() => {
-        dispatch({ type: 'set-error', payload: null });
-      }, 10000);
```

Comment:
this logic clears the errors as webz-wallet might have some during the sync state, but it will keep running anyway. So this logic is still required

Reply:
It cleared WebZjsContext-s ` state.error` which wasn't even used in the app, it was just set and cleared for no reason.

Now its used by react-hot-toast which removes the toast after set duration.